### ISSUE TITLE
utop unable to install on 4.03.0-beta2

### ISSUE THREAD

**Issue by neolem** [diml/utop#165]
I am getting the error below while trying to install utop on `4.03.0-beta2` via opam. The error trace below.

```
Warning 58: no cmx file was found in path for module Ocamlbuild_cppo, and its interface was not compiled with -opaque
### stderr ###
# W: Cannot find source file matching module 'UTop_cmt_lifter' in library utop
# E: Failure("Command ''... .opam/4.03.0+beta2/bin/ocamlbuild' src/lib/utop.cma src/lib/utop.cmxa src/lib/utop.a src/lib/utop.cmxs src/camlp4/utop-camlp4.cma src/camlp4/utop-camlp4.cmxa src/camlp4/utop-camlp4.a src/camlp4/utop-camlp4.cmxs src/top/uTop_top.byte src/top/uTop_top_full.byte -use-ocamlfind -plugin-tags 'package(cppo_ocamlbuild)' -tag debug' terminated with error code 10")
# make: *** [build] Error 1
```

**Comment by diml** [comment]
Can you try the development version (`opam pin add --dev utop`)? I think @whitequark fixed the build with beta2

**Comment by neolem** [comment]
Tried opam pin, got the error below.
```
utop is now git-pinned to git://github.com/diml/utop.git

[utop] git://github.com/diml/utop.git updated
[utop] Installing new package description from git://github.com/diml/utop.git

utop needs to be installed.
Your request can't be satisfied:
  - utop.1.19 is not available because the package is pinned to git://github.com/diml/utop.git, version 1.19.

No solution found, exiting
[NOTE] Pinning command successful, but your installed packages may be out of sync.
```

**Comment by diml** [comment]
hmm,  it looks like a weird opam error, you should report this to opam directly. You can also try to manually clone the utop repo and build it by hand with `make`

**Comment by neolem** [comment]
Yep, tried to manually build utop but even that failed since `oasis` is not yet available for ocaml version `> 4.02`. So luck as yet.

**Comment by diml** [comment]
This oasis situation is frustrating... you can try the branch `release-1.19`, which has the oasis generated stuff committed in

**Comment by neolem** [comment]
This seems to be opam issue. I installed `oasis` using older ocaml `4.02` in the system after which I can locally build `utop` master branch with `4.03.0+beta2`. However, note `opam pin utop ... ` is still failing. Closing this issue now.

**Comment by diml** [comment]
OK, I did a new minor release: https://github.com/ocaml/opam-repository/pull/6259

Thanks for testing

**Comment by yannham** [comment]
Is this supposed to be fixed ? I'm still unable to build utop 1.19.1 for 4.03.0+beta2+flambda or 4.03.0+trunk+flambda with the same error message

**Comment by diml** [comment]
Indeed, replying on #167
